### PR TITLE
[[Bug 13012]] update currentTimeChanged docs

### PR DESCRIPTION
**Pull request by MartinKoob**
Indicate that currentTimeChanged message is sent while player is playing.

### CODE REVIEW

File: docs/dictionary/message/currentTimeChanged.lcdoc [hunk 1]
```diff
@@ -25,12 +25,30 @@ newTime:
...
+	- clicking the play button in the controller bar,
+	- clicking in the controller bar, 
+	- dragging the controller bar scroller thumb,
+	- using the arrow keys when the player is the active (focused)control,
+	- using the scrubber control in the controller bar. 
+		(which is revealed by clicking the arrow buttons in the controller bar
+		 while holding the shift key down.)
+
+<<<<<<< Local Changes
+Changing the player's <currenttime><property> property by script does not trigger the sending of the <currentTimeChanged><message>.
```
Comment (montegoulding) [inline docs/dictionary/message/currentTimeChanged.lcdoc]: @MartinKoob I think your editor must have added these diff markers.

File: docs/dictionary/message/currentTimeChanged.lcdoc [hunk 2]
```diff
@@ -25,12 +25,30 @@ newTime:
...
+a user changes the current time in a player.
+
 Handle the <currentTimeChanged> message if you want to do something when
 the user changes the current time in a <player>.
 
-The user can change the current time by clicking in the controller bar
-or by using the arrow keys when the player is the active (focused)
-control. 
+The user can change the current time by: 
+	- clicking the play button in the controller bar,
```
Comment (montegoulding) [inline docs/dictionary/message/currentTimeChanged.lcdoc]: @MartinKoob The formatting here is a bit wrong. Should be an extra line before the list and then the list shouldn't be indented:

```
The user can change the current time by: 

- clicking the play button in the controller bar,
- clicking in the controller bar, 
- dragging the controller bar scroller thumb,
- using the arrow keys when the player is the active (focused)control,
- using the scrubber control in the controller bar. 
  (which is revealed by clicking the arrow buttons in the controller bar
  while holding the shift key down.)
```

Comment (MartinKoob) [reply docs/dictionary/message/currentTimeChanged.lcdoc]: OK thanks Monte.  Just learning how the doc formatting works.

**Comment by livecodepanos** [comment]
Thank you very much @MartinKoob 

@livecode-vulcan review ok dd1bc59

**Comment by livecode-vulcan** [comment]
💙 review by @livecodepanos ok dd1bc59a67ebf1b0e67abc5d65e535aa0ba6894c

**Comment by livecode-vulcan** [comment]
😎 test success dd1bc59a67ebf1b0e67abc5d65e535aa0ba6894c

* try-community-armv6-android-api8: [success](https://vulcan.livecode.com/builders/try-community-armv6-android-api8/builds/216)
* try-community-armv6-android-api9: [success](https://vulcan.livecode.com/builders/try-community-armv6-android-api9/builds/218)
* try-community-js-emscripten-sdk1.35: [success](https://vulcan.livecode.com/builders/try-community-js-emscripten-sdk1.35/builds/204)
* try-community-universal-ios-iphoneos10.3: [success](https://vulcan.livecode.com/builders/try-community-universal-ios-iphoneos10.3/builds/148)
* try-community-universal-ios-iphonesimulator10.3: [success](https://vulcan.livecode.com/builders/try-community-universal-ios-iphonesimulator10.3/builds/148)
* try-community-universal-mac-macosx10.6: [success](https://vulcan.livecode.com/builders/try-community-universal-mac-macosx10.6/builds/214)
* try-community-universal-mac-macosx10.9: [success](https://vulcan.livecode.com/builders/try-community-universal-mac-macosx10.9/builds/219)
* try-community-x86-linux-debian7: [success](https://vulcan.livecode.com/builders/try-community-x86-linux-debian7/builds/218)
* try-community-x86-linux-debian8: [success](https://vulcan.livecode.com/builders/try-community-x86-linux-debian8/builds/222)
* try-community-x86_64-linux-debian7: [success](https://vulcan.livecode.com/builders/try-community-x86_64-linux-debian7/builds/218)
* try-community-x86_64-linux-debian8: [success](https://vulcan.livecode.com/builders/try-community-x86_64-linux-debian8/builds/222)
* try-community-x86-win32: [success](https://vulcan.livecode.com/builders/try-community-x86-win32/builds/242)
* try-community-x86_64-win32: [success](https://vulcan.livecode.com/builders/try-community-x86_64-win32/builds/212)

**Pull request merged**